.github/workflows/dev-ci-cd.yml

```
@@ -0,0 +1,96 @@
+
+env:
+  REGISTRY: docker.io
+  IMAGE_NAME: ${{ secrets.DOCKER_USERNAME }}/wonderwander
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    outputs:
+      image_tag: ${{ steps.meta.outputs.tags }}
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)